roles/bind-unifiedpush-apb/tasks/main.yml

```diff
@@ -56,4 +74,14 @@
       appType: "{{ CLIENT_TYPE }}"
       clientId: "{{ CLIENT_ID }}"
       isProduction: "{{ iosIsProduction }}"
+      upsInstanceName: "{{ upsInstanceName }}"
   when: CLIENT_TYPE == "IOS" and APB_TEST is not defined
+
+- name: Populate {{ CLIENT_ID }} APB bind credentials
+  set_fact:
+    _apb_bind_creds: "{{ _apb_bind_creds|default({}) | combine( {item.name: item.value} ) }}"
+  with_items:
+    - { name: clientId, value: '{{ CLIENT_ID }}' }
+    - { name: upsInstanceName, value: '{{ upsInstanceName }}' }
+  when: not encode_asb_binding
+  no_log: yes
```

Comment:
what's the purpose of this section here?

Reply:
@wei-lee since [`asb_encode_binding`](https://github.com/aerogearcatalog/unifiedpush-apb/pull/82/files#diff-a332b78855a44dc7568a6d007f9fc7ebL43) module does not work in apb test, we need to populate the bind credentials somehow - this is one of the approaches.